docs/topics/proc_cli-execute.adoc

```diff
@@ -25,56 +27,67 @@ $ ./mta-cli --input /path/to/jee-example-app-1.0.0.ear --output /path/to/output
 
 . Access the report.
 
-[discrete]
 [id="command-examples_{context}"]
-== {ProductShortName} Command examples
+== {ProductShortName} command examples
 
 [discrete]
 === Running {ProductShortName} on an application archive
 
 The following command analyzes the `com.acme` and `org.apache` packages of the link:https://github.com/windup/windup/blob/master/test-files/jee-example-app-1.0.0.ear[jee-example-app-1.0.0.ear] example EAR archive for migrating from JBoss EAP 5 to JBoss EAP 7:
 
+[source,terminal]
 ----
-$ <MTA_HOME>/bin/mta-cli --input /path/to/jee-example-app-1.0.0.ear --output /path/to/report-output/ --source eap:5 --target eap:7 --packages com.acme org.apache
+$ <MTA_HOME>/bin/mta-cli \
+--input /path/to/jee-example-app-1.0.0.ear \
```

Comment:
Indent all but the first line. Please see your Linux documentation on how to break long lines.